app/app.js

```diff
@@ -0,0 +1,56 @@
+import {App, IonicApp, Platform} from 'ionic/ionic';
+
+import {HelloIonicPage} from './pages/hello-ionic/hello-ionic';
+import {ListPage} from './pages/list/list';
+
+@App({
+  templateUrl: 'build/app.html'
+})
+class MyApp {
+  constructor(app: IonicApp, platform: Platform) {
```

Comment:
We pass some parameters into the constructor: `app` and `platform`. These parameters inject the IonicApp and Platform services (which are provided by the Ionic library) into the constructor.

The IonicApp service provides some information about the app, and also keeps track of all the components within the application (as well as allowing us to select specific components).

The Platform service provides information about the platform that the app is running on (e.g. width, height, landscape, portrait etc.), in this case we’re using it to detect when the app is ready.